rest_framework_simplejwt/authentication.py

```diff
@@ -66,6 +66,10 @@ def get_raw_token(self, header):
         """
         parts = header.split()
 
+        if not parts:
```

Comment:
I generally like to make things as explicit as possible.  If we check for the empty case in this way, it makes it more clear that the result is some kind of list of parts that is empty.  Otherwise, it could be any number of different values for which `bool(value) is False`.
```suggestion
        if len(parts) == 0:
```